### DISCUSSION TITLE
Infolist component key/name issue while testing

### DISCUSSION THREAD

**Discussion by shepard153** [filamentphp/filament#11056] [Help]
### Package

Infolist builder

### Package Version

v3.2

### How can we help you?

Hi. I'm currently writing tests for infolist hint actions and I have problem accessing nested components. This is my infolist schema:
```
$infolist->schema([
    Tabs::make('tabs')
        ->key('tabs')
        ->tabs([
            Tabs\Tab::make('notes')
              ->key('notes')
              ->schema([
                 TextEntry::make('internal_note')
                    ->key('internal_note')
                    ->hintAction($this->editInternalNoteAction())
                    ...
              ])
          ])
    ])


// Test:
livewire(OrderDetails::class, ['order' => $this->order])
    ->callInfolistAction('internal_note', 'editInternalNote', data: ['content' => 'test']);
```

The only component I can access is the one on the first level which is just tabs component. I've tried dot notation (`tabs.notes.internal_note`) but still noting. Any idea how can I test these actions?

**Comment by ImChriz** [comment]
Hi, I have the same problem while writing tests. Any updates on this?

**Comment by hfoletto** [comment]
I'm also facing the same issue, has anyone found a solution?
